code/game/machinery/OpTable.dm

```diff
@@ -9,11 +9,12 @@
 	active_power_consumption = 5
 	var/mob/living/carbon/patient
 	var/obj/machinery/computer/operating/computer
-	buckle_lying = -1
+	buckle_lying = TRUE
```

Comment:
would you mind moving the inherited vars above the ones being defined on this type? same with `can_buckle`

Reply:
Addressed